libs/atqr/domain/src/lib/challenge-entity/challenge-entity.test.ts

```diff
@@ -0,0 +1,131 @@
+import { Challenge } from './challenge-entity';
+import { Player } from '../player';
+import { Guid } from '@tokilabs/lang/';
+import { EmailAddress } from '../..';
+import { PaymentMethodEntity, PaymentMethodEnum } from '../payment-method';
+
+// arrange, act and assert
+describe('Challenge', () => {
+  test('Create and call a new challenge', () => {
```

Comment:
All tests contents are PERFECT. But I think that their descriptions could be better. "Create and call a new challenge" does indeed describe what you're doing, but test's descriptions should capture the intention of the code, kind of like a hint to the next programmer, doing that they can assess if the subject of a specific test was indeed met! I'll help you with the first one but the others are up to you. This one could be: "Should correctly be instantiated when given the correct data"